### ISSUE TITLE
New text added is not marked as green

### ISSUE THREAD

**Issue by abdielcs** [jfcherng/php-diff#27]
old file:
X
new file:
XY
z
As the result in demo.php the new text added is not marked as green. 
[old_file.txt](https://github.com/jfcherng/php-diff/files/4423980/old_file.txt)
[new_file.txt](https://github.com/jfcherng/php-diff/files/4423981/new_file.txt)
PHP 7.4.1 , php-diff v6.6.4
Only work on Combined Diff
![php-diff](https://user-images.githubusercontent.com/5825680/78301736-884a1a00-7507-11ea-9f21-dcaddcbd0af5.png)

**Comment by jfcherng** [comment]
Github and many softwares behave like that as well.
https://gist.github.com/jfcherng/96c7297ebef871e17ca87514c58646b7/revisions#diff-1a79a4d60de6718e8e5b326e338ae533

It's expected result. [Unless there are the same amounts of ins/del lines](https://github.com/jfcherng/php-diff/blob/007abaf07ad04e722a30c8163f712a228bbfe375/src/Renderer/Html/AbstractHtml.php#L74-L81), we don't further find their corresponding lines. 

`Combined` renderer doesn't provide line number information so it just treat them like a single line (which just has some `\n`s) and then mark detailed diff.

---

But I am fine that if someone is going to make a PR for this. There is maybe-helpful implementation of the `Combined` renderer.

**Comment by jsandeo** [comment]
I agree that this would be useful to easily spot differences that are otherwise not obvious at first sight depending on the actual diff. For instance, the addition of a blank in a line is not obvious unless it is highlighted in green.

The diff2html Javascript library does mark those differences, as shown here:

<img width="1857" height="413" alt="Image" src="https://github.com/user-attachments/assets/d1ae0376-bb88-43ad-a38b-f86f6097173f" />

**Comment by jfcherng** [comment]
> I agree that this would be useful to easily spot differences that are otherwise not obvious at first sight depending on the actual diff. For instance, the addition of a blank in a line is not obvious unless it is highlighted in green.
> 
> The diff2html Javascript library does mark those differences, as shown here:
> 
> <img alt="Image" width="1857" height="413" src="https://private-user-images.githubusercontent.com/10262516/507088436-d1ae0376-bb88-43ad-a38b-f86f6097173f.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjE3NDIwMzYsIm5iZiI6MTc2MTc0MTczNiwicGF0aCI6Ii8xMDI2MjUxNi81MDcwODg0MzYtZDFhZTAzNzYtYmI4OC00M2FkLWEzOGItZjg2ZjYwOTcxNzNmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTEwMjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUxMDI5VDEyNDIxNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTRkYTI4NjNmYWIyNmRlZjM4YTdiOWMzNmNiNzdmYjc4N2U5N2VmYTNjNWUwNGQ2OWUxNDUwN2MzZTQ1MjdiOGImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.Qp3vxJ1KP9v2gz8ocEEcVaDYFhXLLr4hiNOvpa4wCLo">

I believe we are not talking the case in your screenshot here. This lib behaves like what in your screenshot is.
